src/integrations/prefect-email/prefect_email/credentials.py

```diff
@@ -191,16 +193,21 @@ def example_get_server_flow():
             server = SMTP(smtp_server, smtp_port)
         else:
             context = (
+                ssl.create_default_context() if self.verify else
                 ssl._create_unverified_context(protocol=ssl.PROTOCOL_TLS_CLIENT)
-                if self.unverified_context
-                else ssl.create_default_context()
             )
             if smtp_type == SMTPType.SSL:
                 server = SMTP_SSL(smtp_server, smtp_port, context=context)
             elif smtp_type == SMTPType.STARTTLS:
                 server = SMTP(smtp_server, smtp_port)
                 server.starttls(context=context)
         if self.username is not None:
+            if not self.verify or smtp_type == SMTPType.INSECURE:
+                self._logger.warning(
```

Comment:
We'll want to get the run logger here to ensure it is populated with details of the current run.
```suggestion
                get_run_logger().warning(
```